GVRf/Framework/framework/src/main/jni/objects/components/transform.cpp

```diff
@@ -34,6 +34,9 @@ Transform::Transform() :
 }
 
 Transform::~Transform() {
+    if(owner_object_) {
+        owner_object_->onTransformChanged();
```

Comment:
@NolaDonato What is the purpose of calling onTransformChanged from the dtor?

On a side note the onTransformChanged method violates the component abstraction in SceneObject. There is a need for a general Component-changed mechanism?